.yarn/sdks/typescript/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "typescript",
+  "version": "4.2.3-pnpify",
```

Comment:
I wonder if we should have some step in the CI to keep the version up to date, so it doesn't eventually get more and more misleading